src/Sylius/Bundle/ShopBundle/templates/account/change_password.html.twig

```diff
@@ -20,12 +18,5 @@
 {% endblock %}
 
 {% block account_content %}
-    <div>
-        {{ form_start(form, {'action': path('sylius_shop_account_change_password'), 'attr': {'novalidate': 'novalidate'}}) }}
-
-        {% hook 'sylius_shop.account.change_password' with { form } %}
-
-        {{ form_row(form._token) }}
-        {{ form_end(form, {'render_rest': false}) }}
-    </div>
+    {{ component('sylius_shop:account:change_password:form') }}
```

Comment:
I wonder if it wouldn't be more consistent with other pages if we triggered the hook here and defined this component in the YAML configuration, wdyt?